docs/projects/blank/tutorial/0/key-pieces.md

```diff
@@ -0,0 +1,156 @@
+<%(TOC)%>
+
+# Key pieces
+
+The Blank Project is a barebones SpatialOS project containing the minimal functionality required to begin development with the GDK. It includes:
+
+* A basic Unity project.
+* Local and cloud launch configurations.
+* A default snapshot containing only a single PlayerCreator entity.
+* Client-worker and server-worker configurations.
+* Worker build configuration.
+* Unity Scenes to use in development.
+
+## Clone the project
+
+Before exploring the contents of the Blank Project, we recommend you first download the source code. There are two ways you can do this: _either_ get the source code as one zip file download _or_ clone the repository using Git. (To find out more about Git, see [github.io](https://try.github.io)).
+
+<%(Callout message="We recommend using Git, as Git's version control makes it easier for you to get updates in the future.")%>
+
+<%(#Expandable title="Zip file download")%>
+
+ While we recommend using Git, if you prefer to, you can get the source code for the Blank Project by downloading one zip file <a href="https://github.com/spatialos/gdk-for-unity-blank-project/releases" target="_blank">here</a>. Please download the latest release, the file should be called something like `gdk-for-unity-blank-project-x.y.z.zip`.
+
+<%(/Expandable)%>
+
+<%(#Expandable title="Clone the repository using Git")%>
+
+If you haven't downloaded the zip file, you need the Blank Project repository.
+
+Clone the Blank Project using one of the following commands:
+
+|       |                                                                                |
+| ----- | ------------------------------------------------------------------------------ |
+| HTTPS | `git clone https://github.com/spatialos/gdk-for-unity-blank-project.git` |
+| SSH   | `git clone git@github.com:spatialos/gdk-for-unity-blank-project.git`     |
+
+> You can only clone via SSH if you have already [set up SSH keys (GitHub)](https://help.github.com/articles/connecting-to-github-with-ssh/) with your GitHub account.
+
+<%(/Expandable)%>
+
+## Configurations
+
+At the root of the project you’ll find two launch configurations; one for local deployments and one for cloud deployments.
+
+```text
+
+    gdk-for-unity-blank-project/
+        ├── snapshots/
+        ├── workers/
+        ├── cloud_launch.json
+        ├── default_launch.json
```

Comment:
Dunno if its possible, but it'd be nice to highlight/bold the files that we talk about

Reply:
they said it couldn't be done
![image](https://user-images.githubusercontent.com/10051819/63163212-b6342a00-c01c-11e9-8253-3133eeeacef7.png)